docs/_tutorials/03_extending_wash/writing-an-external-plugin.md

```diff
@@ -38,11 +38,18 @@ function print_entry_json() {
   if [[ -z "${attributes_json}" ]]; then
     attributes_json="{}"
   fi
+
```

Comment:
This page doesn't appear to be formatting correctly in general.

Reply:
I don't understand -- I can see line numbers and code, and copy-pasting the code is pretty easy.

Reply:
Ok, I can't trust github's markdown rendering for this.

Reply:
Ah, gotcha. Yeah, should have checked this out from my master branch then. Apologies for that.